examples/language-summarization/amlt.yaml

```diff
@@ -0,0 +1,42 @@
+description: dp-transformers-examples-bart-summarization
```

Comment:
If we are not including job yaml files, I can remove this.

Reply:
These are Microsoft internal, we'd have to translate them to be compatible with the Azure CLI v2. The syntax is quite similar but not exactly the same. For now feel free to just remove it, we can add them later.